TodoList/TodoList.Api.Tests/Utils/ExtensionItemCompare.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NUnit.Framework.Constraints;
+
+namespace TodoList.Api.Tests.Utils
+{
+    public static class ExtensionItemCompare
```

Comment:
This class contains extensions for `EqualContraint` and should thus be named `EqualContraintExtensions` by .NET custom (that is unless you decided to put multiple `*Constraint` extensions there and all were concentrated on equality, then it could also be called `EqualityExtentions` or something similar).